src/lib/ingest.js

```diff
@@ -134,47 +132,44 @@ function logIngestItemsResults(results) {
         // log it as info and not error
         logger.warn('Invalid ingest item', result.error)
       } else {
-        logger.error('Error while ingesting item', result.error)
+        logger.error('Error while ingesting item::', result.error)
       }
     } else {
       logger.debug('Ingested item %j', result)
     }
   })
 }
 
-export async function ingestItems(items) {
+/* eslint-disable no-await-in-loop */
+
+export async function processMessages(msgs) {
   const results = []
-  for (const record of items) {
-    let dbRecord
+  // apply messages one-at-a-time in sequence
+  for (const msg of msgs) {
+    let dbOp
     let result
     let error
     try {
-      // We are intentionally writing records one at a time in sequence so we
-      // disable this rule
-      // eslint-disable-next-line no-await-in-loop
-      dbRecord = await convertIngestObjectToDbObject(record)
-      // eslint-disable-next-line no-await-in-loop
-      result = await writeRecordToDb(dbRecord)
+      dbOp = await convertIngestMsgToDbOperation(msg)
+      result = await executeDbOperation(dbOp)
     } catch (e) {
       error = e
     }
-    results.push({ record, dbRecord, result, error })
+    results.push({ record: msg, dbRecord: dbOp, result, error })
```

Comment:
preserve the output format of this message, even though we've changed the internal names.